mastering-lightning/README.md

```diff
@@ -112,6 +112,8 @@ https://github.com/lnbook/lnbook
 1. Maintaining a channel graph and performing pathfinding are computationally quite expensive. Could we offload that work to third parties? How would that impact an individual's privacy, security and ability to receive payments?
 1. In which ways does pathfinding rely on messages from the gossip protocol?
 1. Could a malicious actor lie about his channel's fees? How would that affect the network, and how can we assume it's not in his best interest to do so?
+1. Does sending MPP incur more fees compare to sending a single payment if the `base_fee` and `fee_rate` were the same? 
```

Comment:
It's a good starting point, but having equal `base_fee` and `fee_rate` makes it a pretty straightforward/obvious question imo. The interesting part is where `base_fee` and `fee_rate` are not necessarily the same, and nodes can optimize routes. Would extend it a bit.
```suggestion
1. Does sending a Multi-Path Payment (MPP) incur more fees compared to sending a single payment if the `base_fee` and `fee_rate` are the same? Is 
```

Reply:
What do you think about extending the question? Unmarking as resolved.

(Also noticed an additional "Is" sneaked in at the end of my suggestion)

Reply:
Yes, seen it. I want to add a part where they are not the same. I committed so I can get it to my local system. setting up a new system.